src/visualizePath.js

```diff
@@ -0,0 +1,39 @@
+import { PIXELS_PER_TILE } from './constants';
+
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["renderer"] }] */
+function createPathSprites(path, renderer, cpt) {
```

Comment:
Should this function be generalized? Is its only use case to fill in a shortest path with that hard-coded green color? Or could it be useful to call it something like `fillTilesWithColor` and input a generic list of tiles and then a HEX color code?

^ actually a question, not one of those weird suggestions through a question.